06week/checkpoint2.js

```diff
@@ -0,0 +1,103 @@
+
+// Given 1000 digits of PI as strings, return an array of the digits as numbers
+let nums = [];
+const strAsNum = strNums.map(str => nums.push(str));
+console.log(strAsNum);
+// Find the sum of the even values
+let sumEvens = [];
+const evenSum = strNums.reduce((total, currentValue) => currentValue % 2 === 0 ? sumEvens.push(total + currentValue) : total, 0)  
+console.log(evenSum);
+// console.log(sumEvens);
+
+//Find the index of the first "Austin" value (the value plus its index equals 512)
+let atxIdx;
```

Comment:
Don't use let, change to const and assign the return of your find function to the variable